LBH.AdultSocialCare.Api/V1/Controllers/NursingCarePackageController.cs

```diff
@@ -11,9 +12,11 @@
 
 namespace LBH.AdultSocialCare.Api.V1.Controllers
 {
-    [Route("api/v1/nursingCarePackage")]
+    [Route("api/v1/nursing-care-package")]
```

Comment:
I think we should begin just using [controller] instead of route naming for these controllers

Reply:
Doing it that way if you decide to change the controller name in future it affects all existing requests